src/test/util.ts

```diff
@@ -141,6 +150,22 @@ export class TestClient {
     });
   }
 
+  async cleanup() {
+    await new Promise((resolve) => setTimeout(resolve, 1));
```

Comment:
Comment for this function? And why we need the timeout.

Reply:
Done